framework/utils.py

```diff
@@ -32,6 +32,32 @@ def is_internal_ip(ip):
     parsed_ip = IPAddress(ip)
     return parsed_ip.is_private
 
+def get_rank(user_rank, owtf_rank):
```

Comment:
Now I am realizing that we don't need such function. Instead, all we need is the dict translating the numerical ranking to a string.

The `get_rank` function can be removed and instead we could do the following:
```python
# In some util file
from ptp.libptp.constants import INFO, LOW, MEDIUM, HIGH
OWTF_UNRANKED = -1
OWTF_PASSING = 0
OWTF_CRITICAL = 5
RANKS = {
    OWTF_UNRANKED: 'Unranked',
    OWTF_PASSING: 'Passing',
    INFO: 'Informational',
    LOW: 'Low',
    MEDIUM: 'Medium',
    HIGH: 'High',
    OWTF_CRITICAL: 'Critical'
}

# In the API
poutput["rank"] = RANKS.get(max(user_rank, owtf_rank), RANKS[OWTF_UNRANKED])